quora-service/src/main/java/com/upgrad/quora/service/dao/UserDao.java

```diff
@@ -79,10 +77,10 @@ public void updateUserEntity(final UserEntity updatedUserEntity) {
     }
 
     /**
-     * Delete a user by given id from the DB.
+     * Method to delete user by id
      *
-     * @param userId Id of the user whose information is to be fetched.
-     * @return User details which is to be deleted if exist in the DB else null.
+     * @param userId : username which you want to delete
```

Comment:
Some places its 'userid' other places its 'uuid'. Please follow same conventions.

Reply:
Changed userId to uuid.